counterparty-core/counterpartycore/lib/transaction.py

```diff
@@ -102,6 +102,7 @@ def check_transaction_sanity(db, source, tx_info, unsigned_tx_hex, encoding, inp
         if encoding == "p2sh":
             # make_canonical can't determine the address, so we blindly change the desired to the parsed
             desired_source = parsed_source
+            # desired_destination = parsed_destination
```

Comment:
we can remove this, yeah?

Reply:
without this line it's impossible to compose a simple send with p2sh.. in fact the comment for desired_source is valid for destination also..

Reply:
replace send by broadcast and removed this commented line.